docs/query-help-style-guide.md

```diff
@@ -36,7 +36,7 @@ Section-level elements are used to group the information within the query help f
 3.  `example`—an example of code showing the problem. Where possible, this section should also include a solution to the issue.
 4.  `references`—relevant references, such as authoritative sources on language semantics and best practice. 
 
-For further information about the other section-level, block, list and table elements supported by the qhelp format, see [Qhelp files](https://help.semmle.com/wiki/display/SD/Qhelp+files).
+For further information about the other section-level, block, list and table elements supported by the qhelp format, see the [Qhelp reference](https://help.semmle.com/QL/learn-ql/ql/writing-queries/query-help.html) on help.semmle.com.
```

Comment:
Ideally, the link name should be the name of the topic, so this ought to be "Query help reference" and not "Qhelp".